### DISCUSSION TITLE
Is graphql-java hiding null value in the response

### DISCUSSION THREAD

**Discussion by egginbag** [graphql-java/graphql-java#2302] [General]
Based on the GraphQL specification
https://spec.graphql.org/June2018/#sec-Value-Completion

If result is null (or another internal value similar to null such as undefined or NaN), return null.

The PropertyDataFetcher get null value for certain field (let's say name), but in the response the name field is hidden. 

```
{
  "id" : 1,
}
```

I was expecting
```
{
  "id" : 1,
 "name": null
}
```

**Comment by dugenkui03** [comment]
@egginbag 
The result will hold the null value, does the json util you use omit the null value?

**Reply by egginbag** [reply]
this is what I am getting now 
```
{
  "id" : 1,
}
```
seems like the null value is omitted

**Comment by ooga** [comment]
Is there a public repo to take a look ?

**Comment by andimarek** [comment]
I am quite certain this is not a GraphQL java issue. Please have a look at how you are tending the GraphQL java response or how the framework you are using does it (e.g. spring, dgs, Kickstarter etc)
